server/validators/bulk-engage.js

```diff
@@ -0,0 +1,20 @@
+const yup = require('yup');
+
+const BulkEngageParticipantSchema = yup
+  .object()
+  .noUnknown('Unknown field in form')
+  .shape({
+    participants: yup.array().of(yup.number().required()).required(),
```

Comment:
Is the second `required()` necessary? I feel like if we already require array to consist of at least 1 number element that should be sufficient
Also, do you think it'd be nice to provide more descriptive error messages?

Reply:
I will update this... thanks

Reply:
Done